.travis.yml

```diff
@@ -26,33 +22,21 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=4.0.*
+  matrix: SYMFONY_VERSION=4.*
   global:
     - SYMFONY_DEPRECATIONS_HELPER="/.*each.*/"
     - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
     - KERNEL_CLASS=Symfony\Cmf\Bundle\ResourceRestBundle\Tests\Fixtures\App\Kernel
-    - SYMFONY_PHPUNIT_VERSION=5.7
     - TEST_INSTALLATION=false
 
 matrix:
-  include:
-    - php: 7.2
-      env: STABILITY=dev SYMFONY_VERSION=4.0.*
-    - php: 7.1
-      env: STABILITY=dev COMPOSER_FLAGS="--prefer-lowest" SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
please keep the lowest version build, but with php 7.2 

we should be able to remove the SYMFONY_VERSION constant from the line though

Reply:
I don't get it. What do you mean about the lowest version build? I did the lowest version 7.2. Or is there something else?

Reply:
this build sets `COMPOSER_FLAGS="--prefer-lowest"` to have composer install the lowest possible version of everything. this is to detect when we accidentally start using something that is only available in newer versions of our dependencies. then we can either not use it, or raise the minimum version.

the flag only makes sense in reusable libraries like this one. for applications on the other hand, i never bother with that, as we will install the newest possible version of everything anyways.